log4tango/tests/clock.hh

```diff
@@ -34,7 +34,11 @@
     typedef long usec_t;    /* number of microseconds since 1970/01/01 */
 #   define INT64_CONSTANT(val)  (val##L)
 #else
-    typedef int64_t usec_t;
+    #ifdef _TG_WINDOWS_
+        typedef uint64_t usec_t;
+    #else
+        typedef __int64 usec_t;
+    #endif
```

Comment:
Why are you changing the non windows type here form int64_t to __int64? Most POSIX, aka non windows, platforms support C99 so you can use `int64_t` [1]. It also looks really odd to use different signedness in both cases as well.

[1]: https://stackoverflow.com/a/6013417

Reply:
This change is breaking the Linux compilation.
If one looks at the logs of Travis CI builds, one can see there are errors when compiling this file due to this change.